config/settings.yml

```diff
@@ -1105,7 +1105,6 @@
           :queue_timeout: 120.minutes
           :dequeue_method: sql
       :event_handler:
-        :dequeue_method: miq_messaging
```

Comment:
Is this redundant because of the above `messaging.type` change? I can see why part of https://github.com/ManageIQ/manageiq/pull/22198 was reverted/altered but not sure about this line.

Reply:
This was added as part of https://github.com/ManageIQ/manageiq/pull/21997 this isn't a revert because we still want the functionality from that PR but the default settings should change back to before that PR

Reply:
Thanks, I was missing the partial revert/keep.

Reply:
No problem I put this in feeling rushed for time and the PR description reflects that :)